### PR TITLE
ChromeProxyService implements noSuchMethod

### PR DESCRIPTION
**Pull request by elliette**
Have the Chrome proxy service implement `noSuchMethod`, so that VmService changes do not break DWDS at compile time. Without this, DWDS could block Dart SDK rolls into google3.

### CODE REVIEW

**Comment by grouma** [comment]
Can we broaden the dependency on `package:vm_service`? @bkonyi

**Comment by bkonyi** [comment]
> Can we broaden the dependency on `package:vm_service`? @bkonyi

Yeah, I don't think we need to be pinned anymore, so we can have a loose upper bound.

File: dwds/pubspec.yaml [hunk 1]
```diff
@@ -32,8 +32,7 @@ dependencies:
   stack_trace: ^1.10.0
   sse: ^4.1.0
   uuid: '>=2.0.0 <4.0.0'
-  # We pin the version because we implement the interface.
-  vm_service: 7.4.0
+  vm_service: ^1.0.0
```
Comment (bkonyi) [inline dwds/pubspec.yaml:35]: You'll actually want this to be `^7.4.0` :-)

**Review by bkonyi — APPROVED** [review]

File: dwds/pubspec.yaml (hunk 1 above)
Comment (elliette) [reply dwds/pubspec.yaml]: Done !

**Pull request merged**